app/controllers/configs_controller.rb

```diff
@@ -2,6 +2,7 @@ class ConfigsController < ApplicationController
   before_action :confirm_admin_user
 
   def index
+    # n+1 join here
```

Comment:
Can we throw an issue in the backlog like
> `grep "n+1 join here"` and fix em 

while we're at it?

Reply:
yeah! doing so now

Reply:
#2225